### ISSUE TITLE
Global issues with tutorials and examples galleries

### ISSUE THREAD

**Issue by dccowan** [simpeg/simpeg#981]
There are a few global issues related to our example and tutorial galleries that must be addressed:

**Where do we want to store data loaded into tutorials?**

Primarily when creating inversion tutorials, we start by loading a set of supporting files (geophysical data, topography etc...). We need to store these files somewhere. The storage option should:

- Be easily accessible to all SimPEG members with push access to the repository, not just UBC members (so no ownCloud)
- Avoid downloading or storing files in the SimPEG repository. These files may be subject to change which can prompt a committable change
- Ideally be free

**Where do we want to output data created by tutorials?**

Some tutorials have the option to output the results of the forward simulation. If set to output, the files are written to the tutorial directory which is bad. These files need to be written somewhere better.

**Hard cap for maximum run time?**

Do we want to enforce a maximum run time for tutorials or examples?

**Comment by jcapriot** [comment]
These are great points Devon! We should discuss a little more about them at the next meeting. Something else I’d like to discuss is the possibility of separating the tutorials themselves into there own repository. They are getting rather cumbersome in terms of computation time and it might be best to just provide a hyperlink to another separate documentation page that has all of them.

**Comment by domfournier** [comment]
I completely agree with Joe. Tutorials should be its own project enterily, and just referenced as an "external project" on the main page. Examples on the other hand should be extremely light and simply exposing functionality and not use cases. 
This brings back the topic of "What is the mission of" simpeg, which I don't think we, as a group, haven't really  answered yet.
